### PR TITLE
Correcting mishaps + upgrade tombstone

### PR DESCRIPTION
**Pull request by vronc**
* I realized I had crucial changes on the wrong branch for the tombstone I created (the language string was missing + the tombstone was initiated as food made of carpet material..!). :man_facepalming:
This PR fixes that.

* I also made a fun change: The tombstone is now fertilizable if it's on dirt => some small flowers will appear on top of the grave. This will work even if it's on deceased dirt which goes against principles but I thought it would be fun if a corpse could give some new life to the dirt.

### CODE REVIEW

**Comment by Shrimpstress** [comment]
This needs to be updated to use the latest master considering we have recently renamed almost everything in the mod.

**Comment by vronc** [comment]
I thought I pulled it 🤔 still not an expert I guess...

**Comment by Shrimpstress** [comment]
It's no big deal, but one thing you need to keep in mind is that in the latest master `src/main/java/com/fabriccommunity/spookytime` is now `thehallow` at the end. Same for `src/main/resources/assets/spookytime`. SpookyBlocks has been renamed to HallowedBlocks.

**Comment by vronc** [comment]
Should be done now.

**Review by Shnupbups — APPROVED** [review]

File: src/main/java/com/fabriccommunity/thehallow/block/TombstoneBlock.java [hunk 1]
```diff
@@ -47,8 +55,38 @@ public BlockRenderLayer getRenderLayer() {
...
 	public VoxelShape getCollisionShape(BlockState state, BlockView blockView, BlockPos blockPos, EntityContext entityContext) {
 		return SHAPES.get(state.get(FACING));
 	}
+
+	public boolean isFertilizable(BlockView blockView_1, BlockPos blockPos_1, BlockState blockState_1, boolean boolean_1) {
+		return true;
+	}
+
+	public boolean canGrow(World world_1, Random random_1, BlockPos blockPos_1, BlockState blockState_1) {
+		return true;
```
Comment (liach) [inline src/main/java/com/fabriccommunity/thehallow/block/TombstoneBlock.java]: probably should only return true if age is 0

**Comment by LemmaEOF** [comment]
@vronc This is looking pretty good! Just make sure you can't bonemeal a tombstone after it's already grown and this is merge-ready!

**Review by LemmaEOF — APPROVED** [review]

File: src/main/java/com/fabriccommunity/thehallow/block/TombstoneBlock.java [hunk 2]
```diff
@@ -47,8 +55,38 @@ public BlockRenderLayer getRenderLayer() {
...
+		BlockPos down = pos.down();
+		return Block.isFaceFullSquare(world.getBlockState(down).getCollisionShape(world, down), Direction.UP);
+	}
+
 	@Override
 	public VoxelShape getCollisionShape(BlockState state, BlockView blockView, BlockPos blockPos, EntityContext entityContext) {
 		return SHAPES.get(state.get(FACING));
 	}
+
+	public boolean isFertilizable(BlockView blockView_1, BlockPos blockPos_1, BlockState blockState_1, boolean boolean_1) {
```
Comment (Shnupbups) [inline src/main/java/com/fabriccommunity/thehallow/block/TombstoneBlock.java]: Remove `_1` and such from all variable names

**Comment by vronc** [comment]
Done!

**Review by liach — APPROVED** [review]

**Pull request merged**